From ad6e0e2ead75b81bcb1a2d03173c19445adbeb7a Mon Sep 17 00:00:00 2001
From: Bo Yang <bo.yang@amlogic.com>
Date: Tue, 17 Mar 2015 16:51:18 +0800
Subject: [PATCH 5472/5965] PD#104216: dtd: update watchdog default timeout

the watchdog max timeout is 8s.
don't use the default timeout more than 8s.

this commit udpate all dtd of g9tv & g9bb.

Change-Id: I1e3880a7703b93a5e6468fd67d7ddc736f0a5b57
---
 arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd     | 2 +-
 arch/arm/boot/dts/amlogic/mesong9bb_pxp.dtd      | 2 +-
 arch/arm/boot/dts/amlogic/mesong9bb_skt.dtd      | 2 +-
 arch/arm/boot/dts/amlogic/mesong9tv_n210.dtd     | 2 +-
 arch/arm/boot/dts/amlogic/mesong9tv_n300.dtd     | 8 ++++----
 arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd | 8 ++++----
 arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd  | 8 ++++----
 arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd     | 8 ++++----
 arch/arm/boot/dts/amlogic/mesong9tv_pxp.dtd      | 2 +-
 arch/arm/boot/dts/amlogic/mesong9tv_skt.dtd      | 4 ++--
 10 files changed, 23 insertions(+), 23 deletions(-)
 mode change 100755 => 100644 arch/arm/boot/dts/amlogic/mesong9tv_n300.dtd
 mode change 100755 => 100644 arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd
 mode change 100755 => 100644 arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd
 mode change 100755 => 100644 arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
 mode change 100755 => 100644 arch/arm/boot/dts/amlogic/mesong9tv_pxp.dtd
 mode change 100755 => 100644 arch/arm/boot/dts/amlogic/mesong9tv_skt.dtd

diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd
index 0c54bae0117e..022e9e43c773 100644
--- a/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd
@@ -408,7 +408,7 @@
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
 		status = "okay";
-		default_timeout=<10>;
+		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
 		shutdown_timeout=<10>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_pxp.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_pxp.dtd
index 95f6d3294ec5..bf9ac75a8694 100644
--- a/arch/arm/boot/dts/amlogic/mesong9bb_pxp.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9bb_pxp.dtd
@@ -408,7 +408,7 @@
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
 		status = "okay";
-		default_timeout=<10>;
+		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
 		shutdown_timeout=<10>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_skt.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_skt.dtd
index 888cfe4a3624..39b9647fd49f 100644
--- a/arch/arm/boot/dts/amlogic/mesong9bb_skt.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9bb_skt.dtd
@@ -408,7 +408,7 @@
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
 		status = "okay";
-		default_timeout=<10>;
+		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
 		shutdown_timeout=<10>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n210.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n210.dtd
index a549aa68f131..2e531711b42b 100644
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n210.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n210.dtd
@@ -427,7 +427,7 @@ ethmac: ethernet@0xc9410000{
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
 		status = "okay";
-		default_timeout=<10>;
+		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
 		shutdown_timeout=<10>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n300.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n300.dtd
old mode 100755
new mode 100644
index cca2645c23f5..4392574a523b
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n300.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n300.dtd
@@ -420,7 +420,7 @@
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
 		status = "okay";
-		default_timeout=<10>;
+		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
 		shutdown_timeout=<10>;
@@ -543,7 +543,7 @@
 //$$ L3 PROP_U32 = "f_min"
 //$$ L3 PROP_U32 = "f_max"
 //$$ L3 PROP_U32 = "max_req_size"
-//$$ L2 PROP_U32 = "card_type"    
+//$$ L2 PROP_U32 = "card_type"
 		sdio{
 			status = "ok";
 			port = <0>;          /*0:sdio_a, 1:sdio_b, 2:sdio_c, 3:sdhc_a, 4:sdhc_b, 5:sdhc_c */
@@ -1561,7 +1561,7 @@
 		usid-min = <8>; /*reserved*/
 		usid-max = <31>; /*reserved*/
 	};
-		
+
 ///	-	Thermal
 //$$ MODULE="Thermal"
 //$$ DEVICE="thermal"
@@ -1573,7 +1573,7 @@
 		compatible = "amlogic-thermal";
 		#thermal-cells=<7>;
 		dev_name = "aml_thermal";
-        trip_point=<70 1608001 1608001 511 511 3 5 
+        trip_point=<70 1608001 1608001 511 511 3 5
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
                     260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd
old mode 100755
new mode 100644
index c00d387f5bab..225ff1015ac3
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd
@@ -408,7 +408,7 @@
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
 		status = "okay";
-		default_timeout=<10>;
+		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
 		shutdown_timeout=<10>;
@@ -531,7 +531,7 @@
 //$$ L3 PROP_U32 = "f_min"
 //$$ L3 PROP_U32 = "f_max"
 //$$ L3 PROP_U32 = "max_req_size"
-//$$ L2 PROP_U32 = "card_type"    
+//$$ L2 PROP_U32 = "card_type"
 		sdio{
 			status = "ok";
 			port = <0>;          /*0:sdio_a, 1:sdio_b, 2:sdio_c, 3:sdhc_a, 4:sdhc_b, 5:sdhc_c */
@@ -1552,7 +1552,7 @@
 		usid-min = <8>; /*reserved*/
 		usid-max = <31>; /*reserved*/
 	};
-		
+
 ///	-	Thermal
 //$$ MODULE="Thermal"
 //$$ DEVICE="thermal"
@@ -1564,7 +1564,7 @@
 		compatible = "amlogic-thermal";
 		#thermal-cells=<7>;
 		dev_name = "aml_thermal";
-        trip_point=<70 1608001 1608001 511 511 3 5 
+        trip_point=<70 1608001 1608001 511 511 3 5
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
                     260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd
old mode 100755
new mode 100644
index 3d8a5842066a..10b6ec179f32
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd
@@ -408,7 +408,7 @@
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
 		status = "okay";
-		default_timeout=<10>;
+		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
 		shutdown_timeout=<10>;
@@ -531,7 +531,7 @@
 //$$ L3 PROP_U32 = "f_min"
 //$$ L3 PROP_U32 = "f_max"
 //$$ L3 PROP_U32 = "max_req_size"
-//$$ L2 PROP_U32 = "card_type"    
+//$$ L2 PROP_U32 = "card_type"
 		sdio{
 			status = "ok";
 			port = <0>;          /*0:sdio_a, 1:sdio_b, 2:sdio_c, 3:sdhc_a, 4:sdhc_b, 5:sdhc_c */
@@ -1552,7 +1552,7 @@
 		usid-min = <8>; /*reserved*/
 		usid-max = <31>; /*reserved*/
 	};
-		
+
 ///	-	Thermal
 //$$ MODULE="Thermal"
 //$$ DEVICE="thermal"
@@ -1564,7 +1564,7 @@
 		compatible = "amlogic-thermal";
 		#thermal-cells=<7>;
 		dev_name = "aml_thermal";
-        trip_point=<70 1608001 1608001 511 511 3 5 
+        trip_point=<70 1608001 1608001 511 511 3 5
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
                     260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
old mode 100755
new mode 100644
index 67c3fb8b0eac..4d00b5f14dd1
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
@@ -420,7 +420,7 @@
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
 		status = "okay";
-		default_timeout=<10>;
+		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
 		shutdown_timeout=<10>;
@@ -543,7 +543,7 @@
 //$$ L3 PROP_U32 = "f_min"
 //$$ L3 PROP_U32 = "f_max"
 //$$ L3 PROP_U32 = "max_req_size"
-//$$ L2 PROP_U32 = "card_type"    
+//$$ L2 PROP_U32 = "card_type"
 		sdio{
 			status = "ok";
 			port = <0>;          /*0:sdio_a, 1:sdio_b, 2:sdio_c, 3:sdhc_a, 4:sdhc_b, 5:sdhc_c */
@@ -1628,7 +1628,7 @@
 		usid-min = <8>; /*reserved*/
 		usid-max = <31>; /*reserved*/
 	};
-		
+
 ///	-	Thermal
 //$$ MODULE="Thermal"
 //$$ DEVICE="thermal"
@@ -1640,7 +1640,7 @@
 		compatible = "amlogic-thermal";
 		#thermal-cells=<7>;
 		dev_name = "aml_thermal";
-        trip_point=<70 1608001 1608001 511 511 3 5 
+        trip_point=<70 1608001 1608001 511 511 3 5
                     80 1200001 1200001 435 435 2 4
                     90  800001 800001  328 328 2 3
                     260 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff 0xffffffff>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_pxp.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_pxp.dtd
old mode 100755
new mode 100644
index 890d09d7f0d2..20efa37f085c
--- a/arch/arm/boot/dts/amlogic/mesong9tv_pxp.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_pxp.dtd
@@ -215,7 +215,7 @@
 	amlogic-watchdog{
 		compatible = "amlogic,aml-wdt";
 		status = "okay";
-		default_timeout=<10>;
+		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
 		shutdown_timeout=<10>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_skt.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_skt.dtd
old mode 100755
new mode 100644
index 58a9c0bccff4..2b21dcc57665
--- a/arch/arm/boot/dts/amlogic/mesong9tv_skt.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_skt.dtd
@@ -306,7 +306,7 @@
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
 		status = "okay";
-		default_timeout=<10>;
+		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
 		shutdown_timeout=<10>;
@@ -429,7 +429,7 @@
 //$$ L3 PROP_U32 = "f_min"
 //$$ L3 PROP_U32 = "f_max"
 //$$ L3 PROP_U32 = "max_req_size"
-//$$ L2 PROP_U32 = "card_type"    
+//$$ L2 PROP_U32 = "card_type"
 		sdio{
 			status = "ok";
 			port = <0>;          /*0:sdio_a, 1:sdio_b, 2:sdio_c, 3:sdhc_a, 4:sdhc_b, 5:sdhc_c */
-- 
2.19.0

